project_task_type_active/models/project_task_type.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ProjectTaskType(models.Model):
+
+    _inherit = 'project.task.type'
+
+    active = fields.Boolean(default=True)
```

Comment:
no need for a archive/unarchive button (using toggle_active)?

Reply:
Not needed, because there is no form view and Archive/Unarchive actions are available automatically.
![image](https://user-images.githubusercontent.com/778828/41899375-96fbf6a0-792c-11e8-9688-128bdebb15a1.png)